.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": [["es2015", {"loose": true}]],
-  "plugins": ["transform-object-assign"]
+  "presets": ["es3", ["es2015", {"loose": true}]],
+  "plugins": []
```

Comment:
is `transform-object-assign` no longer used? If so, can probably just remove this property altogether.